quickwit/quickwit-indexing/src/actors/indexer.rs

```diff
@@ -388,8 +388,9 @@ impl Actor for Indexer {
             | ActorExitStatus::Failure(_)
             | ActorExitStatus::Panicked => return Ok(()),
             ActorExitStatus::Quit | ActorExitStatus::Success => {
-                self.send_to_serializer(CommitTrigger::NoMoreDocs, ctx)
-                    .await?;
+                let _ = self
```

Comment:
( O_O ) I don't understand the difference?
Are there files missing?

Reply:
In the case where the serializer has already exited, the `send_to_serializer` function will return a `TrySendError::Disconnected` and this will make the `finalizer` fail. The log is quite ugly and I agree with Adrien on this.